shared/scss/popup.scss

```diff
@@ -285,6 +296,9 @@ body {
                             background-image: url($icon__trackers__unblocked--moz);
                         }
 
+                        .is-browser--safari & {
+                            background-image: url($icon__trackers__unblocked--safari);
+                        }
```

Comment:
I'm just going to update everything to use the `cross_platform_bg_image` mixin to make this easier. Lemme do that in a separate PR.